scene/3d/skeleton_3d.cpp

```diff
@@ -639,8 +640,12 @@ int Skeleton3D::get_process_order(int p_idx) {
 	return process_order[p_idx];
 }
 
-void Skeleton3D::localize_rests() {
+const Vector<int> &Skeleton3D::get_process_order() {
```

Comment:
Any idea why `const Vector<int> &` breaks gdscript binding?

Must be `const Vector<int>`.